common/test/acceptance/pages/studio/overview.py

```diff
@@ -236,6 +236,7 @@ def subsection_expanded():
         currently_expanded = subsection_expanded()
 
         self.q(css=self._bounded_selector('.ui-toggle-expansion i')).first.click()
+        self.wait_for_element_presence(self._bounded_selector(self.ADD_BUTTON_SELECTOR), 'Subsection is expanded')
```

Comment:
By default, subsections are collapsed, so this method is used only if someone has to expand a subsection in an acceptance test. I do, however, see your concern.

In theory, this method could be used to collapse a subsection, if called at an inappropriate moment.